jobs/build/scan-images/Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+final MAIL_FROM = 'aos-cd@redhat.com'
+final MAILING_LIST_CVE = ['bbarcaro@redhat.com']
+final MAILING_LIST_ERR = ['bbarcaro@redhat.com']
+
+properties([[
+  $class : 'ParametersDefinitionProperty',
+  parameterDefinitions: [
+    [
+      $class: 'hudson.model.ChoiceParameterDefinition',
+      name: 'BUILD_VERSION',
+      description: 'OCP Version to build',
+      defaultValue: '3.7',
+      choices: ['3.7', '3.6', '3.5', '3.4', '3.3'].join('\n'),
+    ],
+  ],
+]])
+
+node('openshift-build-1') {
+  try {
+    stage('clone') { dir('aos-cd-jobs') { checkout scm } }
+    stage('scan') {
+      sh """\
+aos-cd-jobs/build-scripts/ose_images/ose_images.sh scan_images \
+  --user ocp-build \
+  --branch 'rhaos-${BUILD_VERSION}-rhel-7' \
+  --group base
+cat scan.txt
+"""
+      final ret = sh(
+        returnStatus: true,
+        script: 'grep -xq \'The following issues were found:\' scan.txt')
+      if(ret == 0) {
+        mail(
+          from: MAIL_FROM, to: MAILING_LIST_CVE,
+          subject: 'jobs/build/scan-images: cve(s) found',
+          body: readFile('scan.txt'))
+      }
```

Comment:
Should the job fail at this point or is the email enough?

Reply:
I think email is good. We don't really want to fail on unpatched CVEs because a patch might not even exist yet.

Reply:
After a discussion with @jupierce, we decided it would be better to have this
as a separate job (`build/scan-images`).  So "fail" here means only this
specific job, not the one which built the images.

Reply:
Right - It is ok for the job to fail, but the error must not propagate to the build job.